src/backend/metal/src/lib.rs

```diff
@@ -12,6 +12,7 @@ extern crate core_graphics;
 extern crate block;
 extern crate smallvec;
 extern crate spirv_cross;
+extern crate fxhash;
```

Comment:
why does Metal need this explicitly? I'd assume it to just use the types exported from `hal`

Reply:
You're right, just left it there accidentally